packages/framework-provider-aws/src/library/events-adapter.ts

```diff
@@ -110,3 +108,57 @@ export async function storeEvents(
   await dynamoDB.batchWrite(params).promise()
   logger.debug('[EventsAdapter#storeEvents] EventEnvelope stored')
 }
+
+export async function destroyEntity(
+  dynamoDB: DynamoDB.DocumentClient,
+  config: BoosterConfig,
+  logger: Logger,
+  entityTypeName: string,
+  entityID: UUID
+): Promise<void> {
+  logger.debug('[EventsAdapter#destroyEntity] Destroying entity')
+  // BatchWrite limitation, in the future we should move this value to BoosterConfig. We are assuming that this process will be
+  // fit into lambda memory and time execution limits.
+  const MAX_EVENTS_TO_DELETE = 25
+  let nextKey: DocumentClient.Key | undefined = undefined
+  let result = null
```

Comment:
No need for `null` I believe:
```suggestion
  let result
```

Reply:
Fixed [here](https://github.com/boostercloud/booster/pull/407/commits/a165f7a37c4b48185a08ff610c08f48e7941459e)